File: SpringCloudDev/springcloud-vue-ui/src/entitys/integral/integralstatusaction.js
                                                    export default function area() {
  return {
    current: 1,
    size: 20,
    isNewRecord: false,
    query: {
    // 自增主键
  integralStatusActionId: undefined,
      // 账户号
  accountId: undefined,
      // 客户编号
  custId: undefined,
      // 客户名称
  custName: undefined,
      // 积分类型
  integralType: undefined,
      // 当前余额
  blance: undefined,
      // 操作
  operationType: undefined,
      // 退回原因
  rejectReason: undefined,
      // 状态id
  accountStatusId: undefined,
      // 状态
  accountStatus: undefined,
      // 预留字段一
  reserveColumn1: undefined,
      // 预留字段二
  reserveColumn2: undefined,
      // 预留字段三
  reserveColumn3: undefined,
          },
    rules: {
        integralStatusActionId: [{
  required: true,
  message: '请输入自增主键',
  trigger: 'blur'
  }],
          accountId: [{
  required: true,
  message: '请输入账户号',
  trigger: 'blur'
  }],
          custId: [{
  required: true,
  message: '请输入客户编号',
  trigger: 'blur'
  }],
          custName: [{
  required: true,
  message: '请输入客户名称',
  trigger: 'blur'
  }],
          integralType: [{
  required: true,
  message: '请输入积分类型',
  trigger: 'blur'
  }],
          blance: [{
  required: true,
  message: '请输入当前余额',
  trigger: 'blur'
  }],
          operationType: [{
  required: true,
  message: '请输入操作',
  trigger: 'blur'
  }],
          rejectReason: [{
  required: true,
  message: '请输入退回原因',
  trigger: 'blur'
  }],
          accountStatusId: [{
  required: true,
  message: '请输入状态id',
  trigger: 'blur'
  }],
          accountStatus: [{
  required: true,
  message: '请输入状态',
  trigger: 'blur'
  }],
          reserveColumn1: [{
  required: true,
  message: '请输入预留字段一',
  trigger: 'blur'
  }],
          reserveColumn2: [{
  required: true,
  message: '请输入预留字段二',
  trigger: 'blur'
  }],
          reserveColumn3: [{
  required: true,
  message: '请输入预留字段三',
  trigger: 'blur'
  }],
      }
  }
}

File: SpringCloudDev/springcloud-business-api/springcloud-integral-api/src/main/java/org/springclouddev/integral/entity/IntegralStatusAction.java
package org.springclouddev.integral.entity;

import com.baomidou.mybatisplus.annotation.TableField;
import com.baomidou.mybatisplus.annotation.TableName;
import io.swagger.annotations.ApiModel;
import io.swagger.annotations.ApiModelProperty;
import lombok.Data;
import lombok.EqualsAndHashCode;
import lombok.experimental.Accessors;
import org.springclouddev.core.mp.base.BaseEntity;

/**
 * 积分冻结解冻状态动作表实体类
 *
 * @author zhaobohao
 * @since 2020-01-15
 */
@Data
@Accessors(chain = true)
@TableName("tbl_integral_status_action")
@EqualsAndHashCode(callSuper = true)
@ApiModel(value = "IntegralStatusAction对象", description = "积分冻结解冻状态动作表")
public class IntegralStatusAction extends BaseEntity {

    private static final long serialVersionUID = 1L;

    /**
     * 自增主键
     */
    @ApiModelProperty(value = "自增主键")
    private Long id;
    /**
     * 账户号
     */
    @ApiModelProperty(value = "账户号")
    @TableField("ACCOUNT_ID")
    private String accountId;
    /**
     * 客户编号
     */
    @ApiModelProperty(value = "客户编号")
    @TableField("CUST_ID")
    private String custId;
    /**
     * 客户名称
     */
    @ApiModelProperty(value = "客户名称")
    @TableField("CUST_NAME")
    private String custName;
    /**
     * 积分类型
     */
    @ApiModelProperty(value = "积分类型")
    @TableField("INTEGRAL_TYPE")
    private String integralType;
    /**
     * 当前余额
     */
    @ApiModelProperty(value = "当前余额")
    @TableField("BLANCE")
    private String blance;
    /**
     * 操作
     */
    @ApiModelProperty(value = "操作")
    @TableField("OPERATION_TYPE")
    private String operationType;
    /**
     * 退回原因
     */
    @ApiModelProperty(value = "退回原因")
    @TableField("REJECT_REASON")
    private String rejectReason;
    /**
     * 状态id
     */
    @ApiModelProperty(value = "状态id")
    @TableField("ACCOUNT_STATUS_ID")
    private String accountStatusId;
    /**
     * 状态
     */
    @ApiModelProperty(value = "状态")
    @TableField("ACCOUNT_STATUS")
    private String accountStatus;
    /**
     * 预留字段一
     */
    @ApiModelProperty(value = "预留字段一")
    @TableField("RESERVE_COLUMN1")
    private String reserveColumn1;
    /**
     * 预留字段二
     */
    @ApiModelProperty(value = "预留字段二")
    @TableField("RESERVE_COLUMN2")
    private String reserveColumn2;
    /**
     * 预留字段三
     */
    @ApiModelProperty(value = "预留字段三")
    @TableField("RESERVE_COLUMN3")
    private String reserveColumn3;


}
